client/my-sites/plans/current-plan/jetpack-product-install/index.tsx

```diff
@@ -101,42 +125,72 @@ export class JetpackProductInstall extends Component {
 			return;
 		}
 
-		// Installation can be started only if we have the Akismet and VaultPress keys
-		if ( pluginKeys && pluginKeys.akismet && pluginKeys.vaultpress ) {
-			this.setState( {
-				startedInstallation: true,
-			} );
+		// We need plugin keys to attempt install.
+		if ( ! pluginKeys ) {
+			return;
+		}
+
+		const installerPositionalArguments: ['akismet', 'vaultpress'] = [ 'akismet', 'vaultpress' ];
```

Comment:
Just interesting to note that TypeScript linting/prettier/something doesn't enforce spaces here. 🤷‍♂